service-example/README.md

```diff
@@ -0,0 +1,44 @@
+# Service model example
+
+A service model runs alongside other models. 
+Typically this can be a database which stores information as a workflow progresses. This example contains two models the service and a model which communicates with it, which we have called the client.
+
+Each model contains:
+- a _Dockerfile_ - Builds the model container that will be run by DAFNI
+- a model_definition.yaml_ - A machine-readable file used to define the model.
```

Comment:
Would be good here to note how a service model differs in the model definition from a normal model - highlighting the different type